wiki/osu!_originals/en.md

```diff
@@ -0,0 +1,476 @@
+
+#### ![](/wiki/shared/mode/mania.png) osu! Philippines Nationals 2021
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [NicDroid - Northern Lunar Node](https://www.youtube.com/watch?v=GWltdzuJeJY) | [#1](https://osu.ppy.sh/beatmapsets/1524569) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/mania.png) osu! Philippines Nationals 2022
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Arctic Indexed Systems. - Nightburner](https://soundcloud.com/arcsys04/nightburner) | [#1](https://osu.ppy.sh/beatmapsets/1803916) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/taiko.png) osu! Philippines Nationals 2022 - osu!taiko
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [NicDroid as "NicDono" - TAMBOL2022](https://www.youtube.com/watch?v=zhecFreHC94) | [#1](https://osu.ppy.sh/beatmapsets/1838613) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/osu.png) mezelyus' Summer Tournament 2021
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [SAMString - NUMA](https://soundcloud.com/samstring/numa)[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/1570528) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/osu.png) Resurrection Cup
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Xyris - A Cybernetic-Superluminal-Apocalyptic Post-War Divination](https://soundcloud.com/itsxyris/a-cybernetic-superluminal-apocalyptic-post-war-divination) | [#1](https://osu.ppy.sh/beatmapsets/1814494) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/taiko.png) [Rising Global Taiko Showdown 2021](/wiki/Tournaments/GTS/RGTS_2021)
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Dz'Xa - COSMOBEAT](https://soundcloud.com/dz_xa/cosmobeat) | [#1](https://osu.ppy.sh/beatmapsets/1523659) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/taiko.png) [Squad Global Taiko Showdown 2021](/wiki/Tournaments/GTS/SGTS_2021)
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Ice vs Yukino - R4incarnation](https://soundcloud.com/iceloki/r4incarnation-vs-yukino) | [#1](https://osu.ppy.sh/beatmapsets/1698384) | Grand Finals tiebreaker |
+
+#### ![](/wiki/shared/mode/osu.png) The Perennial 2022
+
+| Song | Beatmap | Notes |
+| :-: | :-: | :-- |
+| [Down - DeviousPanda](https://open.spotify.com/album/7o7flUNX2MptKfLVL4Y2bz) | [#1](https://osu.ppy.sh/beatmapsets/1789325) | Grand Finals NM6 |
+| [UNDEAD CORPORATION - The Throne](https://www.youtube.com/watch?v=PCVtU9kWo1w)[^fa-catalogue] | [#1](https://osu.ppy.sh/beatmapsets/1789376) | Grand Finals tiebreaker |
+
+### Official song remix contest releases
+
+#### [osu! "circles!" Remix Contest](https://osu.ppy.sh/home/news/2016-05-06-osu-circles-remix-contest)
+
+| Song | Notes |
+| :-: | :-- |
+| [BilliumMoto - HDHR](https://soundcloud.com/billiummoto/hdhr) | Winning entry, later featured in [Monthly Beatmapping Contest #13](https://osu.ppy.sh/home/news/2016-12-09-monthly-beatmapping-contest-13-now-open) |
```

Comment:
since it's the only entry with a note, you can turn the note into an inline comment and replace the table with an itemized list

Reply:
i don't think changing this one table will make the article look better as a whole... and i don't think it'll look too bad with osu-web's table styling anyway

Reply:
yes but it's really meaningless on the other hand

Reply:
On your concerns about it becoming outdated, I think you can probably make PR responsibility fall on tournament hosts in case of official/unofficial tournaments, likewise for mapping contest hosts, and individual artists/mappers for standalone maps.

Although that would require that those people know this page exists in the first place and be willing to contribute to it.